msgraph-mail/php/src/ApiClient.php

```diff
@@ -64,7 +64,7 @@ public function __construct(RequestAdapter $requestAdapter) {
     public function usersById(string $id): UserItemRequestBuilder {
         $urlTplParams = $this->pathParameters;
         $urlTplParams['user%2Did'] = $id;
-        return new UserItemRequestBuilder($urlTplParams, $this->requestAdapter);
```

Comment:
I believe this is a regression we've addressed in Go and Python. See
https://github.com/microsoft/kiota/pull/2228
https://github.com/microsoft/kiota/pull/2240

Reply:
Taking  a look